tests/linalg/svd.rs

```diff
@@ -460,3 +460,27 @@ fn svd_sorted() {
         epsilon = 1.0e-5
     );
 }
+
+#[test]
+// Exercises bug reported in issue #983 of nalgebra
+fn svd_consistent() {
```

Comment:
maybe rename to `svd_regression_issue_983` for clarity? Adding a link to the issue might also be nice :)